docs/reference/README.md

```diff
@@ -301,6 +302,36 @@ The wallets below support FIL tokens. We recommend verifying that these have bee
 - A Filecoin light wallet example is in the [Filecoin.js library](https://github.com/filecoin-shipyard/filecoin.js)
 - [Filecoin Rosetta API Proxy](https://github.com/Zondax/rosetta-filecoin) - [Rosetta](https://www.rosetta-api.org/) is an API standard created by Coinbase for a consistent interface to many chains for wallets and exchanges
 
+### Wallet Addresses
+
+When using a wallet, an account is identified by its [address](/about-filecoin/how-filecoin-works.md#addresses). A Filecoin address always starts with the letter `f` and a digit that indicates what type of address it is.
+
+Filecoin accounts have two kinds of address, **public key** addresses, and **ID** addresses. Both addresses refer to the same account and can be used to send and receive FIL using a wallet.
+
+#### Public Key Addresses
+
+A [public key address](/about-filecoin/how-filecoin-works.md#public-key-addresses-f1-and-f3) is derived directly from a cryptographic key. Public key addresses start with the characters `f1` or `f3`, depending on the type of encryption key used.
+
+Here's an example of a secp256k1 public key address: `f1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za`.
+
+Public key addresses are the most common way to refer to Filecoin accounts, and they are supported by hardware wallets like [Ledger](https://ledger.com).
+
+Because a public key address does not depend on any blockchain state, you can use it right away, without waiting for it to be "confirmed". As such, public key addresses are recommended for most use cases involving transfers of FIL, for example, when sending FIL to another user through an exchange.
+
+#### ID Addresses
+
+ID addresses are a compact and more "human friendly" way to refer to an account than public key addresses. ID addresses always start with the characters `f0`, followed by a sequence of digits, for example: `f033259`.
+
+Every ID address for a Filecoin account has an alternative public key address that corresponds to the same account. You can find the ID address for any public key address by searching for the public key address on [FilFox](https://filfox.info/), a Filecoin block explorer.
+
+Because they are more compact than public key addresses, ID addresses are often used when refering to miners and other long-lived Filecoin [Actors](/about-filecoin/how-filecoin-works.md#actors). As these actors receive a large volume of messages, the compact address can result in meaningful savings in gas fees.
+
+While you can send FIL to an ID address using a wallet, it's generally best to use public key addresses for transfers. This is because ID addresses take some time to be finalized on the blockchain, so a brand new ID address might be re-assigned to a different account than the one you intended. Only send FIL to an `f0` address if you've verified that the account is at least a day old. You can check the address on [FilFox](https://filfox.info/) to see when it was created.
```

Comment:
I don't understand the back-end of things, but "at least a day" sounds too open to interpretation or mistakes for my comfort... maybe change the text here (and in the section below) to "only if you've verified the address through a service like FilFox"?

Reply:
(and remove any "at least day" bits)

Reply:
Sounds good. I wasn't feeling great about that either since it's so vague, but didn't trust my knowledge enough to be more precise.

As far as I can tell from digging, the current main net value for "finality" is 900 epochs, which are 30 seconds each, so about ~7.5 hours. I don't think it makes sense to explain that here in detail though... Maybe I could add "finality" to the glossary and put the mainnet value there, then link to the definition from here. Then there's just one place to update the value if it ever changes.

Reply:
👍🏼👍🏼👍🏼 linking to the glossary entry and saying "~7.5hrs but always check first" seems reasonable.